ocaml/testsuite/tests/typing-layouts-err-msg/debug_printer.ml

```diff
@@ -0,0 +1,8 @@
+(* TEST
+   flags = "-extension layouts_alpha"
+   * toplevel
+*)
+type ('a : float64) t = 'a
+let f ppf (x : 'a t) = ();;
+#install_printer f;;
+(* Always shows the "??? has the wrong type for a printing function" message on exception *)
```

Comment:
OK -- then update the text for `Debug_printer_argument` to tell people to give us a ring.